src/scss/nav.scss

```diff
@@ -117,12 +128,9 @@ div.nav-container {
 }
 
 .select-version:focus {
-	border-color: #aaa;
-	box-shadow: #aaa;
-	box-shadow: 0 0 0 3px -moz-mac-focusring;
   color: $mdc-theme-secondary;
```

Comment:
I removed these styles because they were getting overwritten by the next `border-color` statement in this class.